docs/source/operations/indy_verify.md

```diff
@@ -0,0 +1,6 @@
+# Verify that Indy DLT network is working
+1. You need to verifiy that network is successfully configured as mentioned in [setting_dlt.md](https://github.com/hyperledger-labs/blockchain-automation-framework/blob/master/docs/source/operations/setting_dlt.md).
+2. Verify that there are no errors in the logs of stewards.
+2. Connect to cluster using generated pool-genesis. Readme for pool genesis is [here](https://github.com/hyperledger-labs/blockchain-automation-framework/blob/master/platforms/hyperledger-indy/configuration/roles/create/helm_component/pool_genesis/README.md).
+3. Execute multiple commands using CLI for indy-SDK. Readme ho to install and use indy CLI is [here](https://github.com/hyperledger/indy-sdk/tree/master/cli).
```

Comment:
"Readme about how to install and use ... "
This file needs to be added to the operationalguide.rst otherwise this file won't be reachable

Reply:
@tomas-zigardi this item is not fixed. there is a spelling mistake.